getUser/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace getUserInfo
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            User user = new User();
+
+            Console.WriteLine($"Hello stranger, I have a few  questions for you: ");
+
+            user.FirstName = user.AnswerQuestion("What is your first name?");
+            user.LastName = user.AnswerQuestion("What is your last name?");
+            user.FaveAnimal = user.AnswerQuestion("What is your favorite animal?");
+            user.FaveSport = user.AnswerQuestion("What is your favorite sport?");
+            user.FaveNumber = user.AnswerQuestionWithInteger("What is your favorite number?");
+
+            Console.WriteLine($"Hello {user.FirstName} {user.LastName}.");
+            Console.WriteLine($"Your favorite animal is a {user.FaveAnimal}.");
+            Console.WriteLine($"Your favorite sport is {user.FaveSport}.");
+            Console.Write($"And your favorite number is");
```

Comment:
It's fine, but this doesn't have to be string interpolation since there are no variables inside the string